corehq/apps/hqmedia/models.py

```diff
@@ -972,7 +976,7 @@ def get_reference_totals(self):
             Returns a list of totals of each type of media in the application and total matches.
         """
         totals = []
-        for mm in [CommCareImage, CommCareAudio, CommCareVideo]:
+        for mm in [CommCareMultimedia.get_doc_class(t) for t in CommCareMultimedia.get_doc_types()]:
```

Comment:
whats the reason for doing this conversion now?

`CommCareMultimedia` is present in `get_doc_class` but not in `get_doc_types` and that's a little confusing here without the context of why it is skipped.

Reply:
I just did this to dry up the code, so we don't have multiple lists of all media types (excluding CommCareMultimedia because I don't think that ever gets instantiated, it's just a superclass for the others).